### PR TITLE
ZEPPELIN-448 Update documentation for Spark configuration and BASE_PATH change

### PR DESCRIPTION
**Pull request by Leemoonsoo**
https://issues.apache.org/jira/browse/ZEPPELIN-448

Addresses update documentation for Spark configuration.

And BASE_PATH change after #430, #431 is deployed.
(currently deployed website has this change applied)

### CODE REVIEW

File: docs/install/install.md [hunk 1]
```diff
@@ -101,19 +185,14 @@ Configuration can be done by both environment variable(conf/zeppelin-env.sh) and
...
-  </tr>
-  <tr>
-    <td>ZEPPELIN_JAVA_OPTS</td>
-    <td></td>
-    <td>N/A</td>
-    <td>JVM Options</td>
 </table>
 
+<br />
+You'll also need to configure individual interpreter. Informations can be cound in 'Interpreter' section in this documentation.
```
Comment (minahlee) [inline docs/install/install.md]: cound -> found

Comment (felixcheung) [inline docs/install/install.md]: Information

File: docs/interpreter/spark.md [hunk 2]
```diff
@@ -152,18 +194,10 @@ Here are few examples:
...
-		export ZEPPELIN_JAVA_OPTS="-Dspark.jars=/path/mylib1.jar,/path/mylib2.jar -Dspark.files=/path/myfile1.dat,/path/myfile2.dat"
-<br />
-
-
-<a name="zeppelincontext"> </a>
 <br />
 <br />
 ### ZeppelinContext
-
+<hr />
```
Comment (felixcheung) [inline docs/interpreter/spark.md]: is this a typo?

File: docs/interpreter/spark.md [hunk 3]
```diff
@@ -41,10 +41,52 @@ Spark Interpreter group, which consisted of 4 interpreters.
 </table>
 
 
+<br /><br />
+
+### Configuration
+<hr />
+
+Without any configuration, Spark interpreter works out of box in local mode. But if you want to connect to your Spark cluster, you'll need following two simple steps.
```
Comment (felixcheung) [inline docs/interpreter/spark.md]: I think it's important to point out that without setting SPARK_HOME it's running its included version of Spark? (that version depends on the profile used to build Zeppelin)

**Comment by Leemoonsoo** [comment]
@minahlee @felixcheung Thanks for feedback. i pushed new commits and changed this PR's scope to cover BASE_PATH change, too. As a followup of #430, #431

File: docs/_config.yml [hunk 4]
```diff
@@ -55,7 +55,7 @@ JB :
   #   - Only the following values are falsy: ["", null, false]
   #   - When setting BASE_PATH it must be a valid url.
   #     This means always setting the protocol (http|https) or prefixing with "/"
-  BASE_PATH : false
+  BASE_PATH : /docs/0.6.0-incubating-SNAPSHOT
```
Comment (felixcheung) [inline docs/_config.yml]: is there a way to extract out the version? maybe that would be useful to show it on the doc page header like here: http://spark.apache.org/docs/latest/programming-guide.html

**Comment by felixcheung** [comment]
looks good

File: docs/_config.yml (hunk 4 above)
Comment (Leemoonsoo) [inline docs/_config.yml]: There're version number in the doc page header.

https://github.com/Leemoonsoo/incubator-zeppelin/blob/update_install/docs/_includes/themes/zeppelin/_navigation.html#L12

Although it's not using BASE_PATH to display.

both places are documented in docs/README.md to make sure updated when bumping up the version.

**Comment by Leemoonsoo** [comment]
I have rebased to resolve conflict with master branch.
A test added and spark version check in SparkInterpreter, PySparkInterpreter, SparkSQLInterpreter.

Ready to merge.

**Pull request closed without merging**